app/clients/email/aws_ses.py

```diff
@@ -114,6 +132,8 @@ def send_email(self,
             elapsed_time = monotonic() - start_time
             current_app.logger.info("AWS SES request finished in {}".format(elapsed_time))
             self.statsd_client.timing("clients.ses.request-time", elapsed_time)
+            if self.ses_elapsed_time != 0:
```

Comment:
What could cause the case where the elapsed time is 0?

Reply:
don't think we'll get anything since we're already catching all generic `Exception` objects above, but was just an abundance of caution

Reply:
similarly i don't think we'll ever get a `response` object that exists but doesn't have `elapsed`, but better safe than sorry